lib/LessParser.js

```diff
@@ -125,18 +119,19 @@ module.exports = class LessParser extends Parser {
       this.lastNode.mixin = true;
       this.lastNode.raws.identifier = identifier;
 
-      // const importantIndex = tokens.findIndex((t) => importantPattern.test(t[1]));
+      const { params } = this.lastNode;
+      const importantMatch = params.match(importantPattern);
```

Comment:
I think I understand why you're looking at the params to detect `important` here, but I believe that's going to open up the parser to future frailty. particularly when it comes to nested rulesets containing a declaration that has `!important` - e.g. https://github.com/shellscape/postcss-less/blob/master/test/parser/mixins.test.js#L178

the Right Way ™️ to tackle this is by inspecting the tokens

Reply:
You're right, I'll take a closer look at inspecting the tokens.